Sample/UIKit/CustomEventViewController.swift

```diff
@@ -120,7 +89,7 @@ final class CustomEventViewController: UIViewController {
                 assertionFailure("Unexpected view hierarchy")
                 return
             }
-            guard let customAttribute = customAttributeFromData(name: nameTextField.text ?? "",
+            guard let customAttribute = EventHelper.customAttributeFromData(name: nameTextField.text ?? "",
                                                                 value: valueTextField.text ?? "",
                                                                 type: typeTextField.text ?? "") else {
```

Comment:
nit: please adjust indentation of parameters

Reply:
Updated here [ef255d3](https://github.com/rakutentech/ios-inappmessaging/pull/278/commits/ef255d3b5dd56271a86816c6d7f6607538f0e123).

Reply:
Updated here [ef255d3](https://github.com/rakutentech/ios-inappmessaging/pull/278/commits/ef255d3b5dd56271a86816c6d7f6607538f0e123).